docs/guides/getting-started-java-spring.md

```diff
@@ -36,18 +36,22 @@ For example, in this guide we will outline a BPMN model to receive a payment req
 While stepping through this guide, you can visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code to check your work.
 :::
 
-<SmPrereqs/>
+[comment]: SmPrereqs/
 
-## Step 1: Install Camunda 8 Self-Managed
+## Prerequisites
 
-<Install/>
+Before getting started, ensure you have the following in your local environment:
 
-## Step 2: Create a new Spring Boot project
+- Access to your preferred code editor or IDE
+- [OpenJDK 21+](https://jdk.java.net/)
+- [Camunda 8 Desktop Modeler](https://camunda.com/download/modeler/) and [Camunda 8 Server<br/>](https://camunda.com/download/modeler/)_If you've already signed up for a Self-managed account, you can find the download links in your welcome email._
```

Comment:
blocking, in one way or another: 

Previous to this, we've been creating a separate prerequisites sub-document for each different form of installation. I can get on-board with eliminating the sub-documents, especially if this is the final form of this guide. But if this isn't the final expected form of this guide, or the previous practice of keeping these bits in a sub-document resonates with you, we should stick with that approach. 

If we _do_ stick with this approach of pulling this section inline instead of importing from a sub-document, then there are a couple things we should clean up: 

1. The `[comment]: SmPrereqs/` line -- it's unclear to me why we would preserve that path as a "comment". 
2. The now-unused import lines at the top of this file: 
   ```
   import SmPrereqs from './react-components/sm-prerequisites.md'
   import Install from './react-components/install-plain-java.md'
   ```
3. (maybe) The unused sub-documents in [the `react-components` folder](https://github.com/camunda/camunda-docs/tree/4300-update-the-spring-developer-guide-to-include-the-new-installation-of-self-managed/docs/guides/react-components).

Reply:
Thanks for taking a look and sharing your feedback @pepopowitz !

This would be the final form of this guide. Would it be good to add these steps to the separate prerequisites sub-document instead, replacing the older format of installing SM? AFAIK, there are two other guides that use prerequisites and the sub-document would help the other guides as well.

> The [comment]: SmPrereqs/ line -- it's unclear to me why we would preserve that path as a "comment".

This is temporary. If we go ahead with using the sub-document, then this line would get uncommented.

> The now-unused import lines at the top of this file:

These would be removed as well.

PS: This is my first contribution towards docs where I've edited/written using markdown and understanding the structure was new to me :)

Reply:
There is [a component](https://github.com/camunda/camunda-docs/blob/main/docs/guides/react-components/install-c8run.md) for reusing the C8Run installation content created already, but I haven't yet dropped it into existing pages like this. I would recommend using a separate component than the ones that already exist for the Java install, though, as I am not sure each page is ready to use with C8Run.